integrationtests/src/test/java/org/axonframework/integrationtests/commandhandling/AbstractAggregateMemberTest.java

```diff
@@ -0,0 +1,233 @@
+package org.axonframework.integrationtests.commandhandling;
+
+
+import org.axonframework.commandhandling.CommandHandler;
+import org.axonframework.eventsourcing.EventSourcingHandler;
+import org.axonframework.modelling.command.*;
+import org.axonframework.spring.stereotype.Aggregate;
+import org.axonframework.test.aggregate.AggregateTestFixture;
+import org.axonframework.test.aggregate.FixtureConfiguration;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
```

Comment:
A small piece of documentation describing what this class tests right now would be nice.
The test name is rather broad, which on face value doesn't tell you the current intent of the test.
Drafting up one or two sentences explaining the intent will be helpful for future contributors of this code.